glean.1.schema.json

```diff
@@ -276,7 +300,22 @@
         "labeled_rate": {
```

Comment:
A labeled what now?

Reply:
Yeah, the schema has placeholders for a bunch of things that were in the original design doc from 2018.  In hindsight maybe not the greatest.  Maybe we file a separate bug to remove the cruft?